### ISSUE TITLE
Display fiat prices of tokens in mobile wallet

### ISSUE THREAD

**Issue by nop33** [alephium/alephium-frontend#244]

**Comment by ElvisGAN777** [comment]
This is much needed at this point. I am onboarding normies and they are confounded as to why the value of their alternate tokens are not showing up in the total fiat balance.

**Comment by diomark** [comment]
agreed definately good to have this. guessing the prices need to come from alph backend somehow though instead of being fetched from coingecko everytime

**Comment by hopsai** [comment]
It would be nice to have the valuation of the tokens in the wallet as well as the valuation of the ALPH.

**Comment by nop33** [comment]
Thanks for the feedback people. It is our highest priority task. It has been a tricky feature that required work from many teams. It will be released asap.

**Comment by ElvisGAN777** [comment]
Thank you @nop33
